week1/1.5/Notes/Notes/Controllers/Model Controllers/NoteController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  NoteController.swift
+//  Notes
+//
+//  Created by Eric Lanza on 7/6/18.
+//  Copyright © 2018 ETLanza. All rights reserved.
+//
+
+import Foundation
+
+class NoteController: Codable {
+    
+    static let shared = NoteController()
```

Comment:
This is a debatable technique. There's a lot of anti-global variable opinions in the world, but that's a topic for future learning of design pattern.